docs/src/pages/[platform]/getting-started/internationalization/examples/ThemeProviderDirectionExample.tsx

```diff
@@ -0,0 +1,33 @@
+import { Alert, Text, ThemeProvider } from '@aws-amplify/ui-react';
+import { useState } from 'react';
+
+export const ThemeProviderDirectionExample = () => {
+  return (
+    <>
+      <ThemeProvider>
```

Comment:
Have we tested how this behaves if you need to nest `dir` attributes? The Radix documentation only mentions wrapping the entire app, but [using the `dir` attribute on block elements](https://www.w3.org/International/questions/qa-html-dir#basicmarkup) is valid if you need to adjust for content that is translated in a language that is a different direction. This is a little weird use case given we suggest NOT nesting ThemeProvider.

Reply:
That's a good callout. You can nest the ThemeProviders so long as you pass the same theme object to both, but it could be result in unexpected CSS variable cascading issues. Do you think this is a blocker? It feels like a bit of an edge case as I would think you would usually set the directionality for your whole app. And for all the other components other than the ones using `radix`, the dir="rtl" attribute would still work fine. For example for a text translation use case, a TextField or Text component would work correctly.

Is it worth calling out in internationalization that only these components need the direction prop on the ThemeProvider? It felts like too much information, I was trying to keep it simple.